guibot/region.py

```diff
@@ -1033,7 +1034,7 @@ def press_keys(self, keys: str | list[str]) -> "Region":
             self.press_keys(['a', 'b', 3])
         """
         keys_list = self._parse_keys(keys)
-        time.sleep(GlobalConfig.delay_before_keys)
+        time.sleep(self.params["control"]["type_delay"])
```

Comment:
Why was the parameter `delay_before_keys` merged with the parameter `delay_between_keys` given that these mean different things?